library/spdm_requester_lib/libspdm_req_get_digests.c

```diff
@@ -196,7 +196,11 @@ libspdm_return_t libspdm_try_get_digest(void *context, uint8_t *slot_mask,
         sizeof(spdm_context->connection_info.peer_total_digest_buffer),
         spdm_response->digest, digest_size * digest_count);
 
-    spdm_context->connection_info.connection_state = LIBSPDM_CONNECTION_STATE_AFTER_DIGESTS;
+    if (spdm_context->connection_info.connection_state <
+        LIBSPDM_CONNECTION_STATE_AUTHENTICATED) {
```

Comment:
With this, LIBSPDM_CONNECTION_STATE_AFTER_CERTIFICATE will become LIBSPDM_CONNECTION_STATE_AFTER_DIGESTS.

I dont feel good. We should maintain the LIBSPDM_CONNECTION_STATE_AFTER_CERTIFICATE status as well.